src/main/java/com/spm/vasylyshyn/service/UserService.java

```diff
@@ -135,7 +137,7 @@ public ApiResponse addDeviceToUser(Long deviceNumber, CType deviceType, String a
 //                                       String password,
                                        Principal principal) {
         User user = getUserByPrincipal(principal);
-        Device device = deviceService.registerDevice(new DeviceDto(deviceNumber,"",null,address,0L,deviceType,0));
+        Device device = deviceService.registerDevice(new DeviceDto(deviceNumber,"",null,address,0L,deviceType,0,LocalDateTime.now()));
```

Comment:
Fix spaces between commas in the project

Reply:
I just reformat the whole files for that reason

Reply:
Fixed it